### DISCUSSION TITLE
正しく合計金額のデフォルト表示が設定できることを確認するテスト

### DISCUSSION THREAD

**Discussion by chikowaka** [ushmz/subsuke#35] [Product backlog]
誰が　　　：
何をしたい：

テスト要件
前提条件　：
手順　　　：
入力値　　：
期待する値：

**Comment by ushmz** [comment]
実装は `settings_bloc.dart`
https://github.com/ushmz/subsuke/blob/c734f9f5c9e672c855288edea321b673838940bf/lib/blocs/settings_bloc.dart#L93-L106

呼び出し箇所は `config.ios.dart` および `config.android.dart`(予定)
https://github.com/ushmz/subsuke/blob/c62d7496f52b80c30ce9e784960da53b20052095/lib/ui/pages/config/config.ios.dart#L61-L66
